s3watcher/lib/Metrics.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+    create: function (uploadResult) {
+        const metricName = uploadResult.succeeded ?
+            "UploadedImages" : "FailedUploads";
+
+        const dimensions = [{
+            "Name" : "UploadedBy",
+            "Value" : uploadResult.uploadedBy
+        }]
+
+        const timestamp = new Date;
+
+        return {
```

Comment:
Internal application state cannot be shared between executions.

Looks like Cloudwatch Logs have a maximum `POST` rate but there is none specified for `PutMetricData`. 

http://docs.aws.amazon.com/AmazonCloudWatch/latest/DeveloperGuide/cloudwatch_limits.html